src/main/java/analyzer/exercises/twofer/TwoferWalker.java

```diff
@@ -14,8 +12,6 @@ class TwoferWalker implements Consumer<Node> {
     boolean usesIfStatement;
     boolean usesConditional;
```

Comment:
Is `usesConditional` still used somewhere?

Reply:
Nop, you're right, i'm going to delete it.